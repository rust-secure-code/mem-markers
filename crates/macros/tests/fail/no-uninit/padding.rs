use mem_markers::NoUninit;

#[derive(NoUninit)]
struct Foo {
    a: u8,
    b: u16,
}

fn main() {}
