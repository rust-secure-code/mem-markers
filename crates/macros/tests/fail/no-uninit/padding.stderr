error[E0080]: evaluation of constant value failed
 --> $DIR/padding.rs:3:10
  |
3 | #[derive(NoUninit)]
  |          ^^^^^^^^ attempt to divide by zero
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
